From pcihpd-discuss-admin@lists.sourceforge.net Mon May  1 18:59:10 2006
Message-ID: <4456BC22.8090806@jp.fujitsu.com>
From: Kenji Kaneshige <kaneshige.kenji@jp.fujitsu.com>
To: gregkh@suse.de, kristen.c.accardi@intel.com
Cc: Kenji Kaneshige <kaneshige.kenji@jp.fujitsu.com>
Subject: acpi_pcihp: Remove improper error message about OSHP
Date: Tue, 02 May 2006 10:55:46 +0900

This patch removes improper error messages from the acpi_run_oshp()
function.

Signed-off-by: Kenji Kaneshige <kaneshige.kenji@jp.fujitsu.com>
Cc: Kristen Accardi <kristen.c.accardi@intel.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/pci/hotplug/acpi_pcihp.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/pci/hotplug/acpi_pcihp.c
+++ gregkh-2.6/drivers/pci/hotplug/acpi_pcihp.c
@@ -129,7 +129,7 @@ acpi_status acpi_run_oshp(acpi_handle ha
 
 	/* run OSHP */
 	status = acpi_evaluate_object(handle, METHOD_NAME_OSHP, NULL, NULL);
-	if (ACPI_FAILURE(status))
+	if (ACPI_FAILURE(status) && (status != AE_NOT_FOUND))
 		printk(KERN_ERR "%s:%s OSHP fails=0x%x\n", __FUNCTION__,
 			(char *)string.pointer, status);
 	else
